x/stakeibc/types/params.go

```diff
@@ -16,37 +16,39 @@ var (
 	DefaultRewardsInterval        uint64 = 1
 	DefaultRedemptionRateInterval uint64 = 1
 	// you apparently cannot safely encode floats, so we make commission / 100
-	DefaultStrideCommission                 uint64 = 10
-	DefaultValidatorRebalancingThreshold    uint64 = 100 // divide by 10,000, so 100 = 1%
-	DefaultICATimeoutNanos                  uint64 = 600000000000
-	DefaultBufferSize                       uint64 = 5             // 1/5=20% of the epoch
-	DefaultIbcTimeoutBlocks                 uint64 = 300           // 300 blocks ~= 30 minutes
-	DefaultFeeTransferTimeoutNanos          uint64 = 1800000000000 // 30 minutes
-	DefaultSafetyMinRedemptionRateThreshold uint64 = 90            // divide by 100, so 90 = 0.9
-	DefaultSafetyMaxRedemptionRateThreshold uint64 = 150           // divide by 100, so 150 = 1.5
-	DefaultMaxStakeICACallsPerEpoch         uint64 = 100
-	DefaultIBCTransferTimeoutNanos          uint64 = 1800000000000 // 30 minutes
-	DefaultSafetyNumValidators              uint64 = 35
-	DefaultSafetyMaxSlashPercent            uint64 = 10
+	DefaultStrideCommission              uint64 = 10
+	DefaultValidatorRebalancingThreshold uint64 = 100 // divide by 10,000, so 100 = 1%
```

Comment:
We should standardize our units at some points i.e. put everything in bp space or % space